Cocoaheads School_Lesson2.playground/Contents.swift

```diff
@@ -1,27 +1,63 @@
-class Stack {
-    private var array = [Int]()
+/**
+ Class describing a collection of elements in LIFO order.
+ */
```

Comment:
```suggestion
/*
 Class describing a collection of elements in LIFO order.
 */
```
please, look at documentation about multiline comments. https://docs.swift.org/swift-book/LanguageGuide/TheBasics.html

Reply:
@Hannabolotova I used structure 
`/** 
       ...  
*/` accordint to [Apple's documentation](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/ComentBlock.html#//apple_ref/doc/uid/TP40016497-CH103-SW1)